.github/workflows/publish-docker.yml

```diff
@@ -0,0 +1,97 @@
+name: Build and Publish Docker
+
+on:
```

Comment:
Can we make this so I have to manually trigger this workflow and not do scheduled nightly releases. We've been doing rolling releases rather than tagged releases. We just tag it as latest.